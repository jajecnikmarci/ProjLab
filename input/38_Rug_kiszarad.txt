timercontrol enable
starttest
add room room1 3
add item room1 Rug rug
add person room1 Student student1
pickup student1
use student1 rug
printstate room1
endtimer timer1
printstate room1
endtest