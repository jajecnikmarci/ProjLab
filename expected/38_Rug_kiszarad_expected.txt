room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects:
		stunEffect1 
	- lastCleaning: null
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects:
	- lastCleaning: null